src/Libraries/RevitNodes/Elements/FamilyType.cs

```diff
@@ -221,6 +249,145 @@ public static FamilyType ByName(string name)
             };
         }
 
+        /// <summary>
+        /// Create new Family Instance from Geometry (via SAT import)
+        /// </summary>
+        /// <param name="geometry"></param>
+        /// <param name="name"></param>
+        /// <param name="category"></param>
+        /// <param name="templatePath"></param>
+        /// <param name="isVoid"></param>
```

Comment:
Do you think we should have two methods? CreateVoidByGeo and CreateSolid ? sure @kronz would prefer that?